internal/acceptance/alert_test.go

```diff
@@ -0,0 +1,76 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestAccAlert(t *testing.T) {
+	WorkspaceLevel(t, Step{
+		Template: `
+		resource "databricks_sql_query" "this" {
+			data_source_id = "{env.TEST_DEFAULT_WAREHOUSE_DATASOURCE_ID}"
+			name = "tf-{var.RANDOM}"
+			query = "SELECT 1 AS p1, 2 as p2"
+		}
+
+        resource "databricks_permissions" "alert_usage" {
+			sql_alert_id = databricks_alert.alert.id
+			access_control {
+              group_name       = "users"
+              permission_level = "CAN_RUN"
+			}
+		}
```

Comment:
Let's add this as a separate test case in `permissions_test.go` rather than having it in the resource-specific tests, then we can leverage that testing infra to make sure we cover all cases.

Reply:
we can do it there as well.